promotion/drain.go

```diff
@@ -548,26 +548,27 @@ func (service *Service) RedeemAndTransferFunds(ctx context.Context, credentials
 			}
 
 			// perform reputation check for wallet, and error accordingly if there is a reputation failure
-			reputable, cohort, err := service.reputationClient.IsDrainReputable(ctx, walletID, *promotionID, withdrawalAmount)
+			reputable, cohorts, err := service.reputationClient.IsDrainReputable(ctx, walletID, *promotionID, withdrawalAmount)
 			if err != nil {
 				logger.Error().Err(err).Msg("RedeemAndTransferFunds: failed to check reputation of wallet")
 				return nil, errReputationServiceFailure
 			}
 
 			if !reputable {
 				// use the cohort to determine the limit exceeded.
-				switch cohort {
-				case reputation.CohortWithdrawalLimits:
-					// limited withdrawal
-					return nil, errWalletDrainLimitExceeded
-				case reputation.CohortNil:
-					// service failure
-					return nil, errReputationServiceFailure
-				default:
-					return nil, errWalletNotReputable
+				for _, cohort := range cohorts {
```

Comment:
What is the expected behavior if the wallet has a scheduled adaptive captcha at the moment of drain? @Miyayes 

In this code, the wallet will be able to drain the funds (just confirming that's the goal here)